packages/task/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {litProdConfig} from '../../rollup-common.js';
+import {createRequire} from 'module';
+
+export default litProdConfig({
+  packageName: createRequire(import.meta.url)('./package.json').name,
+  entryPoints: ['deep-equals', 'index', 'task'],
+  external: ['@lit/reactive-element'],
```

Comment:
In the future I think we should change all of these to `external: (id) => !(id.startsWith('.') || id.startsWith('/')),`

Reply:
+1